README.md

```diff
@@ -1,108 +1,204 @@
-        }
-    ```
-1. Setup RBAC for tenants (wearing CSP hat)
-    ```yaml
-    apiVersion: rbac.authorization.k8s.io/v1
-    kind: ClusterRole
-    metadata:
-      name: tenant
-    rules:
-    - apiGroups:
-      - api.scs.community
-      resources:
-      - '*'
-      verbs:
-      - '*'
-    ---
-    apiVersion: v1
-    kind: ServiceAccount
-    metadata:
-      name: tenant
-      namespace: tenant-name
-    ---
-    apiVersion: rbac.authorization.k8s.io/v1
-    kind: RoleBinding
-    metadata:
-      name: scs-bind
-      namespace: tenant-name
-    roleRef:
-      apiGroup: rbac.authorization.k8s.io
-      kind: ClusterRole
-      name: tenant
-    subjects:
-    - kind: ServiceAccount
-      name: tenant
-      namespace: tenant-name
-    ```
-1. Create resource (wearing tenant hat, `kubectl --as system:serviceaccount:tenant-name:tenant -n tenant-name`)
-    ```yaml
-    apiVersion: api.scs.community/v1alpha1
-    kind: KeypairV2
-    metadata:
-      name: admin
-      namespace: tenant-name
-    spec:
-      name: admin-keypair
-      publicKey: |-
-        ssh-rsa ...
-    ---
-    apiVersion: api.scs.community/v1alpha1
-    kind: InstanceV2
-    metadata:
-      name: testing-vm
-      namespace: tenant-name
-    spec:
-      name: testing-vm
-      keyPair: admin-keyPair
-      imageName: 'Debian 12'
-      flavorName: 'SCS-1V:1:20'
-    ```
-1. Observe creation of resources
-
-Right now, it would be expected to hand out the `ServiceAccount` token to the actual tenant; When AuthN is done via OIDC (or other means), the `ServiceAccount` `tenant-name/tenant` may be dropped and `RoleBinding` `tenant-name/scs-bind` may point to an actual user/group.
+# SCS Central API
+
+## Premise
+
+By embracing existing open source solutions and bundling them, SCS is intended to
+provide a viable alternative to widely adopted proprietary cloud offerings, including
+Infrastructure-as-a-Service offerings, Kubernetes-as-a-Service offerings and other
+X-as-a-Service offerings.
+
+The choice to embrace existing technology has huge advantages over starting from
+scratch.
+By not reinventing wheels, a lot of effort can be saved. Existing communities can
+be strengthened. The adoption of existing open standards is supported, reducing
+market fragmentation and increasing interoperability.
+
+## Challenge
+
+The challenge: Using popular open source components at cloud service providers
+does not result in a consistent experience for their users, yet.
+
+Each part of the stack is consistent within its own scope: E.g. The
+[OpenStack Networking API](https://docs.openstack.org/api-ref/network/v2/) is sort of
+consistent with the
+[OpenStack Load Balancer API](https://docs.openstack.org/api-ref/load-balancer/v2/).
+
+The OpenStack API's share API idioms like the used AuthN/AuthZ mechanisms. But these
+are not applicable beyond OpenStack services.
+
+Entering general IAM, Keycloak has its own set of API endpoints and authentication
```

Comment:
acronyms like IAM should imho be explained/expanded the first time they are used (or linked to a glossary).

Reply:
Fixed in fe98e9af38e1bad334ffb1d6ae12e4f316cb485d.

Reply:
If possible, we should apply this to the other acronyms as well: 

* AuthN
* AuthZ
* CAPI

If you say "*are basically an abstraction over CAPI/ClusterAPI*", do you mean CAPI and ClusterAPI are two separate things? :wink:

Reply:
> If possible, we should apply this to the other acronyms as well:
> 
> * AuthN
> * AuthZ
> * CAPI

Did that in a66f1ab74a379aa028259957a95c2f2a653629c8. Missed the "acronyms **like**"  the first time around.

> If you say "_are basically an abstraction over CAPI/ClusterAPI_", do you mean CAPI and ClusterAPI are two separate things? 😉

No :D
Fixed that in above commit as well.